mava/systems/tf/maddpg/system.py

```diff
@@ -240,6 +244,7 @@ def __init__(
                 n_step=n_step,
                 sequence_length=sequence_length,
                 period=period,
+                bootstrap_n=bootstrap_n,
```

Comment:
Nice catch @KaleabTessera! So bootstrap_n could not be set before from the system interface?